trex/mini2Dx-sample-trex-runner/core/src/main/java/com/mystudio/ChromeTrexGame/Obstacle.java

```diff
@@ -0,0 +1,63 @@
+package com.mystudio.ChromeTrexGame;
+
+
+import org.mini2Dx.core.Graphics;
+import org.mini2Dx.core.Mdx;
+import org.mini2Dx.core.collision.CollisionBox;
+import org.mini2Dx.core.collision.CollisionPoint;
+import org.mini2Dx.core.graphics.Animation;
+import org.mini2Dx.core.graphics.SpriteSheet;
+import org.mini2Dx.core.graphics.Texture;
+
+import static com.mystudio.ChromeTrexGame.ChromeTrexGame.inGame;
+
+public class Obstacle {
+
+    private float collisionRectHeight;
+    private float collisionRectWidth;
+
+    public CollisionPoint obsPoint = new CollisionPoint();
+    private Physics physics;
+
+
+    public static final String OBSTACLE_LOCATION = "rock.png";
+
+    Texture obstacleTexture = Mdx.graphics.newTexture(Mdx.files.internal(OBSTACLE_LOCATION));
```

Comment:
Fields are not ordered by static final, static, final and regular.

If `obsPoint ` variable never changes reference, please set to final. Also, set to private.